diagnostic.md

```diff
@@ -85,6 +112,11 @@ div {
 
 <!-- your answer starts here -->
 
+Cursive. In order to override inline styles you must use the !important tag in CSS.
+
+ -----Can z-index override inline styles?
+
+
```

Comment:
We just wanted the color and font of the text. The color is yellow and the font is cursive. No need to override anything. And no z-index will not override inline styles, z-index is used to move elements on a 3rd dimensional plane so they can be placed either behind or in front of other objects